app/test_split_integer.py

```diff
@@ -1,21 +1,48 @@
-from app.split_integer import split_integer
-
-
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
-
+import pytest
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
-
-
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+from app.split_integer import split_integer
 
 
-def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+class TestSplitInteger:
+    @pytest.mark.parametrize(
+        "value,number_of_parts,result",
+        [
+            pytest.param(
+                125,
+                5,
+                [25, 25, 25, 25, 25],
+                id=("should split into equal parts "
+                    "when value divisible by parts")
+            ),
+            pytest.param(
+                22,
+                1,
+                [22],
+                id=("should return part equals"
+                    "to value when split into one part")
+            ),
+            pytest.param(
+                19,
+                7,
+                [2, 2, 3, 3, 3, 3, 3],
+                id="should be sorted, when they are not equal"
+            ),
+            pytest.param(
+                6,
+                12,
+                [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1],
+                id="should add zeros when value is less than number of parts"
+            )
+        ]
+    )
+    def test_should_return_correct_values(
+            self,
+            value: int,
+            number_of_parts: int,
+            result: list[int]
+    ) -> None:
+        assert split_integer(value, number_of_parts) == result
 
 
-def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+def test_sum_of_the_parts_should_be_equal_to_value() -> None:
+    assert sum(split_integer(2, 2)) == 2
```

Comment:
to be honest, I don`t think that u need this func here because, for example, when u check  125, 5, [25, 25, 25, 25, 25] and test is passed u already know that sum is correct